packages/starter/example/src/components/SendV0Transaction.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button } from '@mui/material';
+import { useConnection, useWallet } from '@solana/wallet-adapter-react';
+import { AddressLookupTableAccount, AddressLookupTableInstruction, AddressLookupTableProgram, MessageV0, TransactionMessage, TransactionSignature, VersionedTransaction } from '@solana/web3.js';
+import { PublicKey, Transaction, TransactionInstruction } from '@solana/web3.js';
+import type { FC } from 'react';
+import React, { useCallback } from 'react';
+import { useNotify } from './notify';
+
+export const SendV0Transaction: FC = () => {
+    const { connection } = useConnection();
+    const { publicKey, sendTransaction } = useWallet();
+    const notify = useNotify();
+
+    const onClick = useCallback(async () => {
+        if (!publicKey) {
+            notify('error', 'Wallet not connected!');
+            return;
+        }
+
+        let signature: TransactionSignature = '';
+        try {
+            const lookupTable = (await connection.getAddressLookupTable(new PublicKey("F3MfgEJe1TApJiA14nN2m4uAH4EBVrqdBnHeGeSXvQ7B"))).value;
+            if (lookupTable === null) {
+                notify('error', 'Address lookup table wasn\'t found!');
+                return;
+            }
```

Comment:
What about a button that actually creates an LUT at a random address (once they get airdropped, can this be done using a legacy tx?) -- this would demonstrate how to use those instructions and make the example self-contained

Reply:
Yeah fair, I was initially considering that but cut scope. How would you expect LUT creation to be connected to sending a v0 transaction? Maybe local storage?

Reply:
I'd like to skip this in the interest of time. Address lookup tables can be created using the solana cli using `solana address-lookup-table` commands. I added a comment here explaining that.

Reply:
Sounds good!